practice-app/api/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     path('helloworld/', views.HelloWorldView.as_view(), name='helloworld'),
-]
\ No newline at end of file
+    path('paritylist/', views.ParityListView.as_view(), name='paritylist'),
```

Comment:
as the url path, we planned to stick with this structure:
'/parity/' for listing the parities,
'/parity/latest/' for the latest data, '/parity/history' etc. . Hence, you might want to change /paritylist/ for the sake of consistency